lib/controller/accountscontroller.php

```diff
@@ -150,11 +151,20 @@ public function show($accountId) {
 
 	/**
 	 * @NoAdminRequired
+	 *
+	 * @param int $accountId
+	 * @return JSONResponse
 	 */
-	public function update() {
+	public function edit($accountId) {
```

Comment:
I think the method should be named `update` since this is a resource controller.